.env

```
@@ -0,0 +1,9 @@
+DATABASE_URL="postgresql://apaeuser:apaepass@localhost:5432/apaedb"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)